examples/sync_example.ipynb

```diff
@@ -0,0 +1,177 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
Add a title and short description of what this notebook is for. Can take care of this in future PR